From 64a6d1a44dc33ad564be58e7daa3d76fac65772e Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Mon, 31 Jul 2023 16:10:26 +0800
Subject: [PATCH 6/7] Add debian_iso_image_arm64.inc

Add debian_iso_image_arm64.inc from debian_iso_image.inc
and made the following changes:
* fix some packages name: replace amd64 with arm64
* remove packages that failed to build for arm64:
  - octeon-ep, qat, iqvlinux

Test Plan:
PASS: build-pkgs on x86-64 host
PASS: build-image on x86-64 host
PASS: build-pkgs on arm64 host
PASS: build-image on arm64 host
PASS: Deploy AIO-SX on x86-64 target
PASS: Deploy AIO-SX on arm64 target
PASS: Deploy AIO-DX on arm64 targets
PASS: Deploy std(2+2+2) on arm64 targets

Story: 2010739
Task: 48018

Depends-On: https://review.opendev.org/c/starlingx/root/+/889682

Change-Id: I75231696f8a4cec3cc7feeac992e202807576a07
Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 debian_iso_image_arm64.inc | 114 +++++++++++++++++++++++++++++++++++++
 1 file changed, 114 insertions(+)
 create mode 100644 debian_iso_image_arm64.inc

diff --git a/debian_iso_image_arm64.inc b/debian_iso_image_arm64.inc
new file mode 100644
index 0000000..c369bcb
--- /dev/null
+++ b/debian_iso_image_arm64.inc
@@ -0,0 +1,114 @@
+# List of packages to be included/installed in ISO
+
+# bnxt_en - Broadcom kernel modules
+bnxt-en
+bnxt-en-rt
+
+#intel-i40e
+i40e
+i40e-rt
+i40e-cvl-2.54
+i40e-cvl-2.54-rt
+i40e-cvl-4.10
+i40e-cvl-4.10-rt
+
+#intel-iavf
+iavf
+iavf-rt
+iavf-cvl-2.54
+iavf-cvl-2.54-rt
+iavf-cvl-4.10
+iavf-cvl-4.10-rt
+
+#intel-ice
+ice
+ice-rt
+ice-cvl-2.54
+ice-cvl-2.54-rt
+ice-cvl-4.10
+ice-cvl-4.10-rt
+
+# Marvell kernel modules
+#octeon-ep
+#octeon-ep-rt
+
+#intel-igb_uio
+igb-uio
+igb-uio-rt
+
+#initramfs-trigger
+initramfs-trigger
+
+# Intel Quartzville Linux Driver
+#iqvlinux
+#iqvlinux-rt
+
+#intel-opae-fpgaa
+kmod-opae-fpga-driver
+kmod-opae-fpga-driver-rt
+
+# libbnxt_re - Broadcom Infiniband verbs library
+libbnxt-re
+
+#kernel-std
+#bpftool  # not used
+#hyperv-daemons  # not used
+#libcpupower-dev  # not used
+#libcpupower1  # not used
+#linux-compiler-gcc-10-x86
+#linux-config-5.10  # not used
+#linux-cpupower  # not used
+linux-headers-5.10.0-6-arm64
+linux-rt-headers-5.10.0-6-rt-arm64
+linux-headers-5.10.0-6-common
+linux-rt-headers-5.10.0-6-rt-common
+#linux-image-5.10.0-6-arm64-dbg # not used
+#linux-rt-image-5.10.0-6-rt-arm64-dbg # not used
+linux-image-5.10.0-6-arm64-unsigned
+linux-rt-image-5.10.0-6-rt-arm64-unsigned
+#linux-image-arm64-dbg  # not used
+#linux-image-arm64-signed-template  # not used
+linux-kbuild-5.10
+linux-rt-kbuild-5.10
+linux-libc-dev
+#linux-perf-5.10-dbgsym  # not used
+linux-perf-5.10
+#linux-perf  # not used
+#linux-source-5.10 # not used
+#linux-rt-source-5.10 # not used
+#linux-source  # not used
+#linux-support-5.10.0-6  # not used
+#usbip  # not used
+
+#mlnx-ofa_kernel
+mlnx-ofed-kernel-modules
+mlnx-ofed-kernel-modules-rt
+mlnx-ofed-kernel-utils
+
+#qat20
+#qat2.0.l
+#qat2.0.l-rt
+
+#rdma-core
+ibacm
+ibverbs-providers
+ibverbs-utils
+infiniband-diags
+#libibmad-dev  # not used
+libibmad5
+#libibnetdisc-dev  # not used
+libibnetdisc5
+#libibumad-dev  # not used
+libibumad3
+#libibverbs-dev  # not used
+libibverbs1
+#librdmacm-dev  # not used
+librdmacm1
+python3-pyverbs
+rdma-core
+rdmacm-utils
+srptools
+
+#mstflint
+mstflint
+
-- 
2.30.2

